Reworks the migration sections to better showcase Hypershift (#2230)
timescale/docs: use-timescale/migration/index.md

---
title: Migrate your data to Timescale
excerpt: Migrate existing database to Timescale Cloud
products: [cloud]
keywords: [data migration, postgresql, hypershift, RDS]
tags: [ingest, migrate, RDS]
---

# Migrate your data to Timescale Cloud

> If you have data in another database, you can migrate it into Timescale Cloud
> using Timescale Hypershift. You can also use hypershift to migrate your data

Author: ```suggestion
You can migrate data from another database into Timescale Cloud
using Timescale Hypershift. You can also use hypershift to migrate your data
```
